client/apollo/react/src/ContentItemDuoAction/ContentItemDuoActionCommon.tsx

```diff
@@ -0,0 +1,69 @@
+import { ComponentType, ReactElement, useMemo } from "react";
+import { getComponentClassName } from "../utilities/getComponentClassName";
+import { ToggleProps } from "../Toggle/ToggleCommon";
+import { ContentItemProps } from "../ContentItemMono/ContentItemMonoCommon";
+import { ButtonProps } from "../Button/ButtonCommon";
+
+export enum ActionStates {
+  edit = "edit",
+  toggle = "toggle",
+}
```

Comment:
pour éviter tous problème lier au enum et à leur gestion trés étrange avec TS, on recommande de plus export un objet.
```suggestion
export const ActionStates = {
  edit: "edit",
  toggle: "toggle",
} as const
```

Reply:
done :)